ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/YoutubeQualityScriptHandler.swift

```diff
@@ -50,16 +42,8 @@ class YoutubeQualityScriptHandler: NSObject, TabContentScript, TabObserver {
     )
   }()
 
-  static func setEnabled(option: Preferences.Option<String>, for tab: some TabState) {
-    let enabled = canEnableHighQuality(option: option)
-
-    tab.evaluateJavaScript(
-      functionName: "window.__firefox__.\(Self.setQuality)",
-      args: [enabled ? Self.highestQuality : "''"],
-      contentWorld: Self.scriptSandbox,
-      escapeArgs: false,
-      asFunction: true
-    )
+  static func setEnabled(for tab: some TabState) {
+    tab.youtubeQualityTabHelper?.handleConnectionStatusChanged(status: Reachability.shared.status)
   }
```

Comment:
Doesn't seem like there's any point to this method existing in the script handler